### PR TITLE
Add initial production webpack configuration

### PR DESCRIPTION
**Pull request by jjlangholtz**
* Move webpack configuration to a new `config` top level directory.
* Create a new production webpack configuration used in `yarn run build`.
* Install additional devDependencies for webpack/babel
* Remove `client/index.html` and parameterize `client/index.ejs` for
prod/dev use
* Remove inject partials as they are no longer needed

@miq-bot add_label wip, euwe/no, enhancement

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -49,7 +52,7 @@ module.exports = {
       {
         test: /\.html$/,
         use: [
-          'ngtemplate-loader?module=app.core',
+          `ngtemplate-loader?module=app.core&relativeTo=${root}/`,
```
Comment (AllenBW) [inline config/webpack.dev.js]: no longer a string?

Comment (jjlangholtz) [inline config/webpack.dev.js]: Ah still a string, just using [template strings](https://babeljs.io/learn-es2015/#ecmascript-2015-features-template-strings).

Comment (AllenBW) [inline config/webpack.dev.js]: 👁 🙊 😻

**Comment by jjlangholtz** [comment]
@miq-bot remove_label wip

**Comment by chriskacerguis** [comment]
@jjlangholtz what is the PT story link?

**Comment by jjlangholtz** [comment]
https://www.pivotaltracker.com/story/show/137196541

**Pull request merged**